cherryontech/unmatched
# #UM-28 Home Page

## Description
This PR adds the home page components and layout. 


## Related Tickets & Documents
This ticket is working on #UM-28, but hasn't closed it yet 


## Acceptance Criteria

- [ ] The hero section of the homepage must feature a carousel component with 3 slides: "End Imposter Syndrome," "Your Story Matters," and "Connect with STEM Voices." that users can navigate between manually using arrows.
- [x] The homepage must include distinct sections for 'Our Story' and 'Our Mission' with text and images that are clearly laid out for immediate comprehension.
- [x] Present a selection of featured profiles in a grid format, ensuring that each profile card is clearly visible, with uniform size and spacing.
- [ ] The 'Get Resources' CTA button should be centrally located for prominence, encouraging user interaction.
- [ ] All design elements on the homepage should follow a consistent theme using Tailwind CSS for styling, ensuring visual harmony and brand alignment.




## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
| ✓ | :sparkles: New feature     |
| ✓ | 🎨 Style                   |
|  ✓ | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Mobile & Desktop Screenshots/Recordings
![Screenshot 2023-11-20 at 4 54 42 PM](https://github.com/cherryontech/unmatched/assets/115492619/154e0e88-4b52-4bd1-9800-902214ea1000)


## What I learned
Layout is complicated! I'm trying to reuse components for more efficient code and it's a bit tricky to do so- next time I build a React app I will plan out all components and see which can be reusable before building any!

## Testing steps
1. Switch to this branch with the following command: `git switch feature/UM-28-home-page`
2. Run the following scripts: `npm run dev` to start Vite's dev build & `npm run tailwind` to start the Tailwind CSS build
3. View the app in browser (the `npm run dev` script will return the url you can see the app at)
4. Inspect the home page in relation to the Figma design

## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
| ✓  | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
  ![helpplease](https://media.giphy.com/media/jRxG53CYEWk9qG6s15/giphy.gif)
